api/hiveextension/v1beta1/agentclusterinstall_types.go

```diff
@@ -331,6 +335,16 @@ type DiskEncryption struct {
 	TangServers string `json:"tangServers,omitempty" gorm:"type:text"`
 }
 
+type OlmOperator struct {
+	// Enable/disable one or more of the assisted-installer supported operators.
+
+	// The name of the operator
+	Name string `json:"name,omitempty"`
+
+	// Blob of operator-dependent parameters that are required for installation.
+	Properties string `json:"properties,omitempty" gorm:"type:text"`
```

Comment:
I hate that we do this in the REST API, and it's mostly because of the tooling we use that ties the API to the DB schema.  In this case, we didn't want to make new tables and schemas for these properties, but I'd like to have it well-defined in the k8s API at least.

Reply:
> I hate that we do this in the REST API, and it's mostly because of the tooling we use that ties the API to the DB schema. In this case, we didn't want to make new tables and schemas for these properties, but I'd like to have it well-defined in the k8s API at least.

you find me in full agreement on both fronts, the hate and the wish to have it well-defined.